lib/ShareByCircleProvider.php

```diff
@@ -404,12 +404,13 @@ public function move(IShare $share, $recipient): IShare {
 	 * @throws RequestBuilderException
 	 * @throws SingleCircleNotFoundException
 	 */
-	public function getSharesInFolder($userId, Folder $node, $reshares): array {
+	public function getSharesInFolder($userId, Folder $node, $reshares, $shallow = true): array {
```

Comment:
bill ?

can you set variable type for each param ?

Reply:
Ok

Reply:
Incompatible with the interface if I add bool, so I am leaving it like this, sorry :/